app/src/main/java/fr/free/nrw/commons/WelcomePagerAdapter.java

```diff
@@ -54,10 +54,14 @@ public boolean isViewFromObject(View view, Object object) {
     public Object instantiateItem(ViewGroup container, int position) {
         LayoutInflater inflater = LayoutInflater.from(container.getContext());
         ViewGroup layout = (ViewGroup) inflater.inflate(PAGE_LAYOUTS[position], container, false);
-
-        if (position == PAGE_FINAL) {
+        if( BuildConfig.FLAVOR == "beta"){
             ViewHolder holder = new ViewHolder(layout);
             layout.setTag(holder);
+        } else {
```

Comment:
I dont think it should be part of `else`. Please leave the original `if` check intact.

Reply:
In beta flavour we needed to button on each screen of tutorial passed to holder, so I removed if to ensure it is followed in all the layouts inflated. In other versions if condition is still there.